frame/contracts/src/weights.rs

```diff
@@ -234,25 +234,25 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: System Account (r:1 w:1)
 	fn call() -> Weight {
-		(140_234_000 as Weight)
+		(189_970_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
```

Comment:
I don't get it. The weight for a contract call goes up. This should be unrelated to this change. Same across other benchmarking results. There is some static cost added to a contract call. Maybe something merged in the meantime without a benchmark?